### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by GreifswalderNet** [nextcloud/android#7457]
************ CAUSE OF ERROR ************

android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{2691f5b u0 com.nextcloud.client/.media.PlayerService}
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2188)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:237)
	at android.app.ActivityThread.main(ActivityThread.java:8167)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:496)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1100)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: samsung
Device: beyond0
Model: SM-G970F
Product: beyond0lteeea

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: G970FXXU9DTJA

**Comment by axheli** [comment]
I hope this is just another symptom of the same cause, so it will be fixed with #7340

**Comment by axheli** [comment]
@tobiasKaminsky  Can you close this since its a dup

**Comment by GreifswalderNet** [comment]
Wo ist jetzt die Lösung?
Wann kommt ein Update diesbezüglich?
Der ganze Programmierkram ist mir als Nutzer irgendwie egal.

**Comment by axheli** [comment]
@GreifswalderNet Can you please comment in English. Thank you

**Comment by AndyScherzinger** [comment]
Fix has been published to Google Play's beta Channel as 3.14.1 RC1, so you can join the beta program as a tester via the store to get it right away within the next hours or you wait for final release.

**Comment by axheli** [comment]
@AndyScherzinger  I get this error when deleting a picture within the photos tab 

************ CAUSE OF ERROR ************

android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{496cd9c u0 com.nextcloud.client/.media.PlayerService}
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2133)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:245)
	at android.app.ActivityThread.main(ActivityThread.java:7953)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:631)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140151
Build flavor: generic

************ DEVICE INFORMATION ************
Brand: OnePlus
Device: OnePlus8Pro
Model: IN2025
Id: RP1A.201005.001
Product: OnePlus8Pro

************ FIRMWARE ************
SDK: 30
Release: 11
Incremental: 2011010100

**Comment by axheli** [comment]
And I'm on 3.14.1 RC1

**Comment by AndyScherzinger** [comment]
Oh, dear.... 🙁

@tobiasKaminsky issue seems to persist even with your fix 🙄

**Comment by axheli** [comment]
> Oh, dear.... 🙁
> 
> @tobiasKaminsky issue seems to persist even with your fix 🙄

Sorry... :(

**Comment by AndyScherzinger** [comment]
Nothing to be sorry for @axheli, the sooner we know it isn't working the sooner we can search for the root cause 👍

**Comment by axheli** [comment]
> Nothing to be sorry for @axheli, the sooner we know it isn't working the sooner we can search for the root cause 👍

You got it let me know and I will test!

**Comment by InfamousUser** [comment]
Sorry to poop on a party but I also have this crash still on 3.14.1 RC1. It didn't seem to happen on the #7340 QA app, just freezes or I just wasn't as lucky as with main app. Might be imagining it but main menu seems to be snappier.

**Comment by InfamousUser** [comment]
************ CAUSE OF ERROR ************

android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{d2f14f0 u0 com.nextcloud.client/com.owncloud.android.files.services.FileUploader}
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1945)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7356)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)

**Comment by tobiasKaminsky** [comment]
> android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{d2f14f0 u0 com.nextcloud.client/com.owncloud.android.files.services.FileUploader}

@InfamousUser this is something else, this seems to happen due to uploading a file.

The other issues are related to 

> android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{496cd9c u0 com.nextcloud.client/.media.PlayerService}

as you can see it is about PlayerService.

**Comment by tobiasKaminsky** [comment]
Found a way to prevent this crash.
Hopefully it is the last one.

However, I will open up another ticket to discuss it more in detail.

**Comment by InfamousUser** [comment]
Where do I report next error after #7461 test app?

Here it is here, give me a link if you want it elsewhere, sorry I don't know how to make ot fixed height and scrollable, also don't see upload button on mobile:

`  ---- 24 Nov 2020 16:54:37 ----

11-24 15:38:29.300 11223 11223 E AndroidRuntime: Process: com.nextcloud.android.qa, PID: 11223

11-24 15:38:29.300 11223 11223 E AndroidRuntime: android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{728d29d u0 com.nextcloud.android.qa/com.owncloud.android.files.services.FileUploader}

11-24 16:44:55.151  1345  3609 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity bnds=[237,337][439,590]} from uid 10060

11-24 16:44:55.213  1345  1392 I ActivityManager: Start proc 19473:com.nextcloud.android.qa/u0a311 for activity {com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity}

11-24 16:44:55.214  3852  3854 I StorageRedirectNative: process started (logcat): pid=19473, uid=10311, user=0, package=com.nextcloud.android.qa, type=activity

11-24 16:44:55.987 19473 19473 I XLua.Xposed: Loaded com.nextcloud.android.qa:10311

11-24 16:44:56.534 19473 19473 I XLua.XParam: Get setting com.nextcloud.android.qa:10311 value.serial=null

11-24 16:44:56.705   634   634 E Layer   : [Surface(name=AppWindowToken{db353c7 token=Token{eee6106 ActivityRecord{4ea6fe1 u0 com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity t3744}}})/@0x4cc6d53 - animation-leash#0] No local sync point found

11-24 16:44:56.705   634   634 E Layer   : [Surface(name=AppWindowToken{db353c7 token=Token{eee6106 ActivityRecord{4ea6fe1 u0 com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity t3744}}})/@0x4cc6d53 - animation-leash#0] No local sync point found

11-24 16:44:56.907  1345  1383 W ActivityTaskManager: Activity top resumed state loss timeout for ActivityRecord{4ea6fe1 u0 com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity t3744}

11-24 16:44:56.910  1345  1383 W ActivityTaskManager: Activity pause timeout for ActivityRecord{4ea6fe1 u0 com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity t3744}

11-24 16:44:58.333  1345  2171 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity bnds=[237,337][439,590]} from uid 10060

11-24 16:44:58.337  1345  1384 W WindowManager: TaskSnapshotSurface.create: Failed to find top fullscreen for task={taskId=3744 appTokens=[AppWindowToken{db353c7 token=Token{eee6106 ActivityRecord{4ea6fe1 u0 com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity t3744}}}] mdr=false}

11-24 16:45:07.915  1345  1383 W ActivityTaskManager: Activity stop timeout for ActivityRecord{4ea6fe1 u0 com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity t3744}

11-24 16:45:07.915  1345  1383 I ActivityTaskManager: Activity reported stop, but no longer stopping: ActivityRecord{4ea6fe1 u0 com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity t3744}

11-24 16:45:14.194  1345  1345 W JobServiceContext: Time-out while trying to bind c268baf #u0a311/941 com.nextcloud.android.qa/androidx.work.impl.background.systemjob.SystemJobService, dropping.

11-24 16:45:14.202  1345  1345 W JobServiceContext: Time-out while trying to bind c5a33cb #u0a311/608 com.nextcloud.android.qa/androidx.work.impl.background.systemjob.SystemJobService, dropping.

11-24 16:45:14.205  1345  1345 W JobServiceContext: Time-out while trying to bind af2da66 #u0a311/818 com.nextcloud.android.qa/androidx.work.impl.background.systemjob.SystemJobService, dropping.

11-24 16:45:15.699  1345  1345 W JobServiceContext: Time-out while trying to bind 9f1dc6d #u0a311/943 com.nextcloud.android.qa/androidx.work.impl.background.systemjob.SystemJobService, dropping.

11-24 16:45:40.054  1345  1383 W ActivityTaskManager: Activity top resumed state loss timeout for ActivityRecord{4ea6fe1 u0 com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity t3744}

11-24 16:45:40.055  1345  1383 W ActivityTaskManager: Activity pause timeout for ActivityRecord{4ea6fe1 u0 com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity t3744}

11-24 16:45:48.197  1345  1345 W JobServiceContext: Time-out while trying to bind 2d015e #u0a311/945 com.nextcloud.android.qa/androidx.work.impl.background.systemjob.SystemJobService, dropping.

11-24 16:45:50.315  1345  1391 W ActivityManager: Timeout executing service: ServiceRecord{cf5a66c u0 com.nextcloud.android.qa/com.owncloud.android.services.OperationsService}

11-24 16:45:51.067  1345  1383 W ActivityTaskManager: Activity stop timeout for ActivityRecord{4ea6fe1 u0 com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity t3744}

11-24 16:46:05.946  1345 10886 I ActivityTaskManager: Activity reported stop, but no longer stopping: ActivityRecord{4ea6fe1 u0 com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity t3744}

11-24 16:46:07.683  1345  3606 I ActivityTaskManager: Activity reported stop, but no longer stopping: ActivityRecord{4ea6fe1 u0 com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity t3744}

11-24 16:46:20.647  1345  1391 E ActivityManager: ANR in com.nextcloud.android.qa

11-24 16:46:20.647  1345  1391 E ActivityManager: Reason: executing service com.nextcloud.android.qa/com.owncloud.android.services.OperationsService

11-24 16:46:20.647  1345  1391 E ActivityManager:   108% 19473/com.nextcloud.android.qa: 87% user + 21% kernel / faults: 131819 minor 7358 major

11-24 16:46:25.771  1345  3597 E InputDispatcher: Window handle Window{396fe18 u0 Application Not Responding: com.nextcloud.android.qa} has no registered input channel

11-24 16:53:27.699  1345  1391 I ActivityManager: Killing 19473:com.nextcloud.android.qa/u0a311 (adj 900): excessive cpu 228960 during 300080 dur=294175 limit=25

11-24 16:53:27.749  1345  1538 W InputDispatcher: channel 'de92abc com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity (server)' ~ Consumer closed input channel or an error occurred.  events=0x9

11-24 16:53:27.749  1345  1538 E InputDispatcher: channel 'de92abc com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity (server)' ~ Channel is unrecoverably broken and will be disposed!

11-24 16:53:27.754  1345  4439 I WindowManager: WIN DEATH: Window{de92abc u0 com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity}

11-24 16:53:27.755  1345  4439 W InputDispatcher: Attempted to unregister already unregistered input channel 'de92abc com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity (server)'

11-24 16:53:27.768  1345  3606 W ActivityTaskManager: Force removing ActivityRecord{4ea6fe1 u0 com.nextcloud.android.qa/com.owncloud.android.ui.activity.FileDisplayActivity t3744}: app died, no saved state

---- 24 Nov 2020 16:54:37 ----
 `
